pkg/orchestrator/pod.go

```diff
@@ -67,17 +64,49 @@ func ProcessPodList(podList []*v1.Pod, groupID int32, hostName string, clusterID
 
 		// scrub & generate YAML
 		if cfg.IsScrubbingEnabled {
-			for c := 0; c < len(podList[p].Spec.Containers); c++ {
-				redact.ScrubContainer(&podList[p].Spec.Containers[c], cfg.Scrubber)
+			for c := 0; c < len(p.Spec.Containers); c++ {
+				redact.ScrubContainer(&p.Spec.Containers[c], cfg.Scrubber)
 			}
-			for c := 0; c < len(podList[p].Spec.InitContainers); c++ {
-				redact.ScrubContainer(&podList[p].Spec.InitContainers[c], cfg.Scrubber)
+			for c := 0; c < len(p.Spec.InitContainers); c++ {
+				redact.ScrubContainer(&p.Spec.InitContainers[c], cfg.Scrubber)
 			}
 		}
 
+		// The resource version field collected from the Kubelet can't be
```

Comment:
Is there a reason we compute the hash after scrubbing the containers? If possible it would be great if we could do it before scrubbing them 🤔

Reply:
I'd guess that if you have scrubbing enabled or disabled it would yield the same resource version.
Just thinking of the case where this setting is changed and we would send different payloads with the same resource version in it.